configure.ac

```diff
@@ -356,6 +356,7 @@ SR_DRIVER([SCPI PPS], [scpi-pps])
 SR_DRIVER([serial DMM], [serial-dmm], [serial_comm])
 SR_DRIVER([serial LCR], [serial-lcr], [serial_comm])
 SR_DRIVER([Siglent SDS], [siglent-sds])
+SR_DRIVER([Sipeed Slogic Analyzer], [sipeed-slogic-analyzer])
```

Comment:
Not a libsigrok developer, but shouldn't a dependency to libusb be added here ?

Reply:
I can confirm, this is needed to prevent it from trying to build and fail if libusb is not available

Reply:
You are right, libusb should be added.
Thanks :)